File: submissions/abc/110/B.js
'use strict'

const main = (input) => {
    input = input.split('\n')
    const X = parseInt(input[0].split(' ')[2])
    const Y = parseInt(input[0].split(' ')[3])
    const Xs = input[1].split(' ').map(i => parseInt(i))
    const Ys = input[2].split(' ').map(i => parseInt(i))

    const MAX_X = Math.max(...Xs)
    const MIN_Y = Math.min(...Ys)

    for(let i = X + 1; i <= Y; i++) { 
        if(MAX_X < i && i <= MIN_Y){
            console.log('No War')
            return
        }
    }
    console.log('War')
}

main(require('fs').readFileSync('/dev/stdin', 'utf-8'))

File: submissions/abc/110/B.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int N, M, X, Y;
    cin >> N >> M >> X >> Y;
    vector<int> x(N), y(M);
    for(int i = 0; i < N; i++){
        cin >> x.at(i);
    }
    for(int i = 0; i < M; i++){
        cin >> y.at(i);
    }
    int maxX = *max_element(x.begin(), x.end());
    int minY = *min_element(y.begin(), y.end());

    for(int i = X + 1; i <= Y; i++){
        if(maxX < i && i <= minY){
            cout << "No War" << endl;
            return 0;
        }
    }

    cout << "War" << endl;
    return 0;
}